### DISCUSSION TITLE
Webflow landing page to Next.js App

### DISCUSSION THREAD

**Discussion by scottschindler** [vercel/community#773] [Help] [answered]
Hi, 

I have a domain that I own, and it's currently deployed to webflow; let's call it www.example.com. When a user clicks on either login or signup, I want them routed to my next.js app to the domain www.example.com/signup or www.example.com/login. 

How do I do this? When I deploy my next.js app, the root would overwrite the webflow domain. 

Thanks!

**Comment by amyegan** [comment, marked as answer]
If I understand correctly, it seems like you want to be able to use the same URL on two separate services. I believe that a URL rewrite is what you need. I'm not sure how to accomplish it with Webflow. Vercel does have the option to [configure a rewrite](https://vercel.com/docs/project-configuration#project-configuration/rewrites) in your Project Configuration file, but the domain would need to be added to Vercel and redirect to some other domain for your Webflow part. Probably not the ideal solution.

Another option is to use a subdomain, like [account.example.com/signup](), if you are able to use modify your DNS settings to connect the subdomain to Vercel. This would probably be the cleaner solution.

I'm sure there are other ways to handle this that I'm not aware of. If anyone else has ideas, please share them!
